### PR TITLE
EasyGas

### PR DESCRIPTION
**Pull request by sanlee42**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: sources/Account.move [hunk 1]
```diff
@@ -196,7 +198,6 @@ module Account {
         let signer_cap = SignerCapability {addr: Token::token_address<STC>() };
         signer_cap
     }
-
     #[test_only]
     public fun get_genesis_capability_for_test(): SignerCapability {
```
Comment (nkysg) [inline sources/Account.move:202]: 这个函数是不是不用了

File: sources/Account.move [hunk 2]
```diff
@@ -196,7 +198,6 @@ module Account {
         let signer_cap = SignerCapability {addr: Token::token_address<STC>() };
```
Comment (nkysg) [inline sources/Account.move:198]: 这里看注释，StarcoinDAO还用吗

Comment (sanlee42) [reply sources/Account.move:198]: starcoinDAO 没有用了，注释得修改下，不过其他地方用了。

File: sources/EasyGas.move [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        sender: &signer,
+        account_address: address,
+        module_name: vector<u8>,
+        struct_name: vector<u8>,
+        data_source: address,
+    ) acquires GasTokenEntry {
+        CoreAddresses::assert_genesis_address(sender);
+        let genesis_account = GenesisSignerCapability::get_genesis_signer();
+        let gas_token_entry = GasTokenEntry { account_address, module_name, struct_name, data_source };
+        if (exists<GasTokenEntry>(address_of(&genesis_account))) {
```
[the remark this answers is not in the document]
Comment (sanlee42) [reply sources/EasyGas.move:74]: 是的，目前只支持STAR,未来可以升级成vec.

File: sources/Account.move (hunk 2 above)
Comment (sanlee42) [reply sources/Account.move:198]: 不用了，其他模块用，注释fix一下得。

**Comment by nkysg** [comment]
need update https://github.com/starcoinorg/starcoin-framework/blob/main/scripts/dev_setup.sh MPM_VERSION=v1.13.7  @sanlee42

**Review by nkysg — APPROVED** [review]

**Pull request merged**